libowfat not building

hi @danwrt

```
socket/socket_remote4.c:11:23: fatal error: havescope.h: No such file or directory
 #include "havescope.h"
                       ^
compilation terminated.
make[4]: *** [socket_remote4.o] Error 1
make[4]: Leaving directory `/tmp/openwrt/build_dir/target-mips_34kc_uClibc-0.9.33.2/libowfat-0.28'
make[3]: *** [/tmp/openwrt/build_dir/target-mips_34kc_uClibc-0.9.33.2/libowfat-0.28/.built] Error 2
```

https://aur.archlinux.org/packages/libowfat/?comments=all

also last version is 0.29

ps: i'm using a mock to build (http://fedoraproject.org/wiki/Projects/Mock), so it's maybe just a missing dependency/host dependency

--- a/libs/libowfat/patches/001-fixbuild.patch
+++ b/libs/libowfat/patches/001-fixbuild.patch
@@ -1,0 +1,11 @@
+--- a/GNUmakefile
++++ b/GNUmakefile
+@@ -278,6 +278,8 @@ socket_accept4.o socket_accept6.o socket
+ socket_local6.o socket_recv4.o socket_recv6.o socket_remote4.o \
+ socket_remote6.o: havesl.h
+ 
++socket_remote4.o: havescope.h
++
+ dns_nd6.o fmt_xlong.o scan_xlong.o fmt_ip6_flat.o $(TEXTCODE_OBJS): haveinline.h
+ 
+ iob_send.o scan_ip6if.o: havealloca.h

